CONTRIBUTING.md

```diff
@@ -75,16 +75,20 @@ In your local repository:
   as previous section requires. You may find this
   [Re-sync Fork With Upstream guide](https://gist.github.com/OleksiyRudenko/8b3ddb664308de0634b53c525e551d8b)
   helpful to fix the issue.
-- ask your peers for review in
-  [FE p2p chat](https://t.me/joinchat/???????????????),
+
+
+### Codereview
+- you will **require** approvals* from **at least two peers**, so seek
+for sufficient support. 
+  - ask your peers for review in [FE Students chat](https://web.telegram.org/#/im?p=s1495296464_549300875144563897),
   post a link to your PR as well
+  - if in some reason there are not reviwers from p2p course, you can ask your friends outside of the course<br>
+- **you should provide a review** for this same task to **at least 2 PR's** of other students of p2p course
```

Comment:
IMHO, this point is still discussable. The main question is whether a student is able to suggest improvements to a task the student has not actually completed yet?
My assumption is that a student that have a PR re certain task approved can bring new learnings further to other students, which means that a student can provide quality feedback to those who are completing an earlier tasks only.

Putting it other way round the suggestion is
```suggestion
- **you have to provide reviews** on the same task to **at least 2 PRs** of other students of p2p course once your PR is merged, using your learnings from merged PR code review
```

Hope the idea behind is clear.